app/javascript/components/downloads/Downloads.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import axios from 'axios';
+import { PropTypes } from 'prop-types';
+import { Button } from 'react-bootstrap';
+import reportError from '../util/ReportError';
+import { ToastContainer, toast } from 'react-toastify';
+
+class Downloads extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  exportDownloaded = () => {
+    axios.defaults.headers.common['X-CSRF-Token'] = this.props.authenticity_token;
+    axios({
+      method: 'post',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      url: `${window.BASE_PATH}/export/downloaded`,
+      data: { id: this.props.download.id },
+    })
+      .then(() => {
+        toast.success('This download has been marked for removal.', {
+          position: toast.POSITION.TOP_CENTER,
+        });
+      })
+      .catch(err => {
+        reportError(err);
+      });
+  };
+
+  render() {
+    return (
+      <React.Fragment>
+        {this.props.error && (
+          <div>
+            The download link is either invalid, has expired, or the file has already been accessed. Please try exporting again. If you think you received this
+            message in error, please contact the help desk.
+          </div>
+        )}
+        {!this.props.error && (
+          <div>
+            <p>
+              Please note that this export is one-time use only. The system will delete an export file once you&apos;ve downloaded that file. These downloads
+              will be invalid if you attempt another export of this type before retrieving the file(s). Exports will not work if forwarded to another user. You
+              must be logged into Sara Alert to access exports.
```

Comment:
Is it actually more restrictive in that you have to be logged in as the user that requested the download to be able to download it?

Reply:
I'll submit these wording changes to the PO team and get some feedback

Reply:
Has that behavior actually changed from previous? This wording is a direct copy from the current wording.

Reply:
I started a thread with the PO team to help with this.